### PR TITLE
31 fix the page for mobile format

### LINKED ISSUE #31: Fix the page for mobile format
The page is not adapted to be watched in a mobile phone. We have to change the CSS in order to fix this.

### PR DESCRIPTION
**Pull request by pablotolosa-aircury**
Fixed the troubles related with the mobile version of the page.
- Images are now centered with  their caption.
- Titles of the web page are now centered.
- The button menu is now visible and adapted to the mobile version.

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
   - title: "Contact us"
-    url: '#contact'
-docs:
-  - title: "List of projects"
-    children:
-      - title: "Project 1"
-        urL: '#project1'
-      - title: "Project 2"
-        urL: '#project2'
+    url: '#contact'
```
Comment (cristobaldiaz-aircury) [inline _data/navigation.yml]: Missing empty line as end of file

Comment (cristobaldiaz-aircury) [reply _data/navigation.yml]: Also some `url` variables are between '' while others don't. Please use the same format with all of them

File: _includes/masthead.html [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
+        <button class="greedy-nav__toggle hidden" type="button">
+          <span class="visually-hidden">{{ site.data.ui-text[site.locale].menu_label | default: "Toggle menu" }}</span>
+          <div class="navicon"></div>
+        </button>
+        <ul class="hidden-links hidden"></ul>
+      </nav>
     </div>
-</div>
+  </div>
+</div>
```
Comment (cristobaldiaz-aircury) [inline _includes/masthead.html]: Missing empty line as end of file.

File: assets/custom.css [hunk 3]
```diff
@@ -79,23 +79,26 @@
...
-    width: 800px;
+    width: 600px;
+    height: 300px;
+    border-radius: 10px;
     display: block;
     margin: 0 auto;
-    max-width: 100%;
-    height: auto;
-    border-radius: 10px;
+
```
Comment (cristobaldiaz-aircury) [inline assets/custom.css]: Don't add unnecessary empty lines

File: previous-projects.md [hunk 4]
```diff
@@ -12,25 +12,19 @@ intro:
 
 {% include feature_row id="intro" type="center" %}
 <link rel="stylesheet" href="./assets/custom.css">
-<div class="container-projects">
```
Comment (cristobaldiaz-aircury) [inline previous-projects.md:15]: Any reason for removing this?

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]

File: previous-projects.md (hunk 4 above)
Comment (pablotolosa-aircury) [reply previous-projects.md:15]: It was an unused class that doesn´t exist anymore

File: assets/custom.css [hunk 5]
```diff
@@ -79,23 +70,30 @@
     }
 }
 .img-pr{
-    width: 800px;
+    width: 75vw;
```
Comment (cristobaldiaz-aircury) [inline assets/custom.css]: It seems that this solution only works for one of the images in one of the views:
image
image

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]

**Review by cristobaldiaz-aircury — APPROVED** [review]

**Pull request merged**